src/ExpressionEvaluator/Core/Test/ExpressionCompiler/ExpressionCompilerTestHelpers.cs

```diff
@@ -348,6 +335,53 @@ internal static ISymUnmanagedReader ConstructSymReaderWithImports(byte[] exeByte
             }
         }
 
+        internal static readonly MetadataReference InstrinsicAssembly = GetIntrinsicAssembly();
+
+        private static MetadataReference GetIntrinsicAssembly()
+        {
+            var source =
```

Comment:
Is this metadata image going to be embedded as a resource in one of the debugger dlls?  If so, can we pull it from there?  On the one hand, it's nice to have source here, but also seems like it might be better to ensure that our tests don't get out of sync with *reality*...